tasks.py

```diff
@@ -104,10 +104,52 @@ def addTask(queues, func, *args, **kwargs):
             logging.exception("Could not enqueue the task")
     except:
         logging.exception("Could not enqueue the task")
-    
+
     raise ndb.Return(success)
 
 
+class _DeferredTaskEntity(ndb.Model):
+    """Datastore representation of a deferred task.
+
+    This is used in cases when the deferred task is too big to be included as
+    payload with the task queue entry.
+    """
+    data = ndb.BlobProperty(required=True)
+
+
+def run_from_datastore(key_url_safe_id):
```

Comment:
this function could be marked as internal
```suggestion
def _run_from_datastore(key_url_safe_id):
```